paddle/fluid/operators/prefetch_op.cc

```diff
@@ -71,7 +71,7 @@ class PrefetchOpMaker : public framework::OpProtoAndCheckerMaker {
               "(RPCClient) The RPC client object which will be"
               "initialized at most once.");
     AddOutput("Out",
-              "(SelectedRows) result "
+              "(LoDTensor) result "
```

Comment:
I think the type of Output variable is `SelectedRows`, just because the shape was not a certain value.